source/index.html.md

```diff
@@ -0,0 +1,183 @@
+---
+title: Algorithmia API Documentation
+language_tabs:
+  # Must be one of https://git.io/vQNgJ
+  - shell: cURL
+  - python: Python
+includes:
+  - algorithms
+  - algorithm_builds
+  - algorithm_versions
+  - data
+  - organizations
+  - organization_members
+  - scms
+  - users 
+search: true
+---
+# Introduction
+
+Welcome to Algorithmia's API documentation. Here you will find examples of how to manage algorithms, data, and admin resources using Algorithmia's REST API and Python client library.
+
+If this is your first time trying Algorithmia, you might find it useful to start with our [Getting Started guide](/getting-started), which can be found alongside many other resources in the [Algorithmia Developer Center](/developers).
+
+## Base URL
+
+```python
+import Algorithmia
+
+# When passing a custom API endpoint to our Python client, you 
+# may omit the `v1` path prefix, as it will be supplied automatically.
+client = Algorithmia.client('API_KEY', 'https://api.example.com')
+```
+
+By default, we provide examples using the `https://api.algorithmia.com/v1` base URL, but if you are using an enterprise version of Algorithmia hosted under a different domain, you should ensure you modify any code samples as necessary.
+
+Per our [versioning scheme](#versioning), ensure that you supply the appropriate version prefix in addition to the REST API endpoint you wish to use. Thus, if your Algorithmia instance was hosted at `example.com`, your base URL would be `https://api.example.com/v1`. If using a client library, ensure you properly pass your custom endpoint as a parameter when initializing the client.
+
+## Authentication
```

Comment:
:bulb: Let's also be sure to mention that users need to enable algorithm management if they wish to use endpoints beyond `/v1algo/`.